src/openMVG/calibration/exportData.cpp

```diff
@@ -45,7 +49,15 @@ void exportImages(openMVG::dataio::FeedProvider& feed,
 
     openMVG::cameras::UndistortImage(inputImage, &camera, outputImage, openMVG::image::BLACK);
     const boost::filesystem::path imagePath = boost::filesystem::path(debugFolder) / (std::to_string(currentFrame) + suffix);
-    const bool exportStatus = openMVG::image::WriteImage(imagePath.c_str(), outputImage);
+
+#ifndef _WIN32
+	std::string nativePath = imagePath.native();
+#else
+	std::wstring_convert<boost::filesystem::path::codecvt_type> wsConv(&imagePath.codecvt());
+	std::string nativePath = wsConv.to_bytes(imagePath.native());
+#endif
+
+    const bool exportStatus = openMVG::image::WriteImage(nativePath.c_str(), outputImage);
```

Comment:
Oh, you're right. I've just double-checked what `string()` returns and it indeed is always `std::string`. I agree with your fix, it's much simpler. (In fact, `string()` on Windows does something similar to my long version.)